geometry_msgs/Pose[] waypoints
moveit_msgs/CollisionObject[] collision_objects
moveit_msgs/Constraints constraints
---
bool completed
